mock/py/mockbuild/buildroot.py

```diff
@@ -866,3 +866,27 @@ def delete(self):
     @property
     def uses_bootstrap_image(self):
         return self.is_bootstrap and self.use_bootstrap_image
+
+    @traceLog()
+    def install_as_root(self, *deps):
+        """Figure out deps from srpm. Call package manager to install them"""
```

Comment:
I don't get this comment :-/ it seems like become-root wrapper around self.install() only?